# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/antitracking/ContentBlockingNotifier.h
# Commit: 312b4abacc2d
# Full Hash: 312b4abacc2dfe0ab2014d62ca271ac54af6704d
# Author: Dimi Lee <dlee@mozilla.com>
# Date: 2020-04-23 21:43:09
# Regressor Bug: 1616775
# File Overlap Count: 1
# Description:
#   Bug 1616775 - P7. Add IPC for ContentBlockingNotifier::OnDecision r=timhuang,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71016
# ==============================================================================

diff -r 9be5ae499d84 -r 312b4abacc2d toolkit/components/antitracking/ContentBlockingNotifier.h
--- a/toolkit/components/antitracking/ContentBlockingNotifier.h	Thu Apr 23 14:25:31 2020 +0000
+++ b/toolkit/components/antitracking/ContentBlockingNotifier.h	Thu Apr 23 14:25:33 2020 +0000
@@ -17,6 +17,9 @@
 class nsPIDOMWindowOuter;
 
 namespace mozilla {
+namespace dom {
+class BrowsingContext;
+}  // namespace dom
 
 class ContentBlockingNotifier final {
  public:
@@ -49,6 +52,9 @@
   static void OnDecision(nsPIDOMWindowInner* aWindow,
                          BlockingDecision aDecision, uint32_t aRejectedReason);
 
+  static void OnDecision(dom::BrowsingContext* aBrowsingContext,
+                         BlockingDecision aDecision, uint32_t aRejectedReason);
+
   static void OnEvent(nsIChannel* aChannel, uint32_t aRejectedReason);
 
   static void OnEvent(
